### DISCUSSION TITLE
Do I really need to use PHOTOPRISM_SITE_URL?

### DISCUSSION THREAD

**Discussion by gerroon** [photoprism/photoprism#3463] [Q&A] [answered]
Hi

I want to use this internally on my LAN. Do I still need to set a URL?

**Comment by lastzero** [comment, marked as answer]
It is used to generate the sharing URL of albums and to enable HTTPS. If you don't use HTTPS and share links, you don't need to set `PHOTOPRISM_SITE_URL` (unless we add new features in a future release that also use it).
